### PR TITLE
AAP-31326: Add OrgName to the Marketing report

### PR DESCRIPTION
**Pull request by goneri**

### CODE REVIEW

File: ansible_ai_connect/users/reports/generators.py [hunk 1]
```diff
@@ -125,6 +125,7 @@ def generate(
                 "First name",
                 "Last name",
                 "Email",
+                "Organization name",
```
Comment (manstis) [inline ansible_ai_connect/users/reports/generators.py:128]: I imagine having the "Organisation name" column next to "Org ID" may have made Marty's life easier.

**Review by manstis — APPROVED** [review]
LGTM :+1: 

Personally, I'd have placed the new column after "Org ID"; but there's nothing in the JIRA.

**Comment by manstis** [comment]
Vulnerabilities scan fix can be found here: https://github.com/ansible/ansible-ai-connect-service/pull/1303

File: ansible_ai_connect/users/reports/generators.py (hunk 1 above)
Comment (goneri) [reply ansible_ai_connect/users/reports/generators.py:128]: I was following the same order we've got for the other report.

**Review by manstis — APPROVED** [review]
LGTM 👍

**Pull request merged**